models/melo2024onto-educational/metadata.yaml

```diff
@@ -0,0 +1,31 @@
+title: Ontologia para Integração de Dados de Evasão Escolar, Desempenho Acadêmico e Concessão de Auxílios Socioeconômicos
+acronym: Onto-Educacional
+issued: 2024
+modified:
+contributor:
+ - https://dblp.org/pid/368/9900
+ - https://dblp.org/pid/121/5314
+keyword:
+ - school dropout
+ - academic performance
+ - socioeconomic aid
+ - data integration
+ - education
+theme: Class H - Social Sciences
+editorialNote: 
+ontologyType:
+ - domain
+language: pt-br
+designedForTask:
+ - conceptual clarification
+ - information retrieval
```

Comment:
Are you sure information retrieval is correct here? It seems it is mainly for interoperability. 

In the paper, the authors state that "This article presents how to develop a domain ontology to organize knowledge related to school dropout, academic performance and distribution of socioeconomic aid in a Federal Higher Education Institution, as well as integrating data from such constructs."

Reply:
This information was provided by the authors of the ontology. IMHO, we should keep it as it is. If you disagree, please let me know and I can remove it.